rest/client/client.go

```diff
@@ -329,3 +312,27 @@ func (c *communicatorImpl) makeSender(ctx context.Context, td TaskData, opts []L
 
 	return send.NewConfiguredMultiSender(senders...), nil
 }
+
+func (c *communicatorImpl) createCedarGRPCConn(ctx context.Context) error {
+	if c.cedarGRPCClient != nil {
```

Comment:
style nit: can you do `if c.cedarGRPCClient == nil {` so we only have one `return nil` statement.